samples/src/source/tasks/ContentTask.bs

```diff
@@ -0,0 +1,29 @@
+import "pkg:/source/roku_modules/log/LogMixin.brs"
+import "pkg:/source/roku_modules/maestro/core/Utils.brs"
+import "pkg:/source/roku_modules/maestro/core/Request.brs"
+import "pkg:/source/roku_modules/maestro/core/MIOC.brs"
+
+@task("ContentTask", "Task")
+@strict
+class ContentTask
+  private log = new log.Logger("ContentTask")
+
+  @inject("styleManager")
+  private styleManager
+
+  ' @injectClass("mc.net.Request")
+  private request = new mc.net.Request()
+
+  private config = mc.getPath(m.styleManager@.getStyle("app"), "server", {})
+
+  private function execute(args as mc.types.assocarray)
+    command = lcase(mc.getPath(args, "command", ""))
+    result = { isOK: false }
+    ' need to integrate the api stuff
+    result = m.request.get(`${mc.config.endpoint}/movies.json`)
```

Comment:
I don't understand this line. what is `mc.config.endpoint`?  - surely this is a compile error, right?

Reply:
Of course in fact I don't know how to implement it